spec/fixtures/responses/whois.nic.club/club/status_registered.txt

```diff
@@ -0,0 +1,96 @@
+[Querying whois.nic.club]
```

Comment:
Please use the tool `./utils/mkwhois.rb` to generate the fixtures. These debug notes should not be part of the output.